spring-cloud-netflix-core/pom.xml

```diff
@@ -180,6 +185,11 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-security</artifactId>
```

Comment:
You are right, I didn't realized `spring-cloud-commons`was already referencing the spring security libs. Thanks for pointing.